.prow.yaml

```diff
@@ -77,7 +77,7 @@ presubmits:
             - make e2e-test
           env:
             - name: KIND_NODE_IMAGE
-              value: "kindest/node:v1.17.0@sha256:9512edae126da271b66b990b6fff768fbb7cd786c7d39e86bdf55906352fdf62"
+              value: "kindest/node:v1.17.5@sha256:ab3f9e6ec5ad8840eeb1f76c89bb7948c77bbf76bcebe1a8b59790b8ae9a283a"
```

Comment:
Please also change the prow tasks names if you change the version here

Reply:
done

Reply:
done